packages/legacy/core/App/components/inputs/PINInput.tsx

```diff
@@ -29,46 +29,56 @@ const PINInput: React.FC<PINInputProps & React.RefAttributes<HTMLInputElement |
     })
     const { t } = useTranslation()
     const { TextTheme, PINInputTheme } = useTheme()
+    const cellHeight = 48
 
     const style = StyleSheet.create({
-      codeField: {
+      container: {
         flexDirection: 'row',
-        justifyContent: 'space-between',
+        justifyContent: 'flex-start',
         alignItems: 'flex-end',
         flexWrap: 'wrap',
+        flex: 1,
         marginBottom: 24,
       },
-      cell: {
-        width: 40,
-        height: 48,
-        backgroundColor: PINInputTheme.cell.backgroundColor,
```

Comment:
Will this be removed from the theme file if it's unused?

Reply:
It's still being used on the new line 66